8_bit_pic.py

```diff
@@ -1,7 +1,33 @@
+"""
+
+Microchip PICmicro MCU implementation.
+
+Specifics:
+- Harvard architecture - separate program and data memories
+- 21-bit program address - up to 2MB program memory (flash)
+- 12 to 16-bit instruction bus
+- big-endian instruction words
+- 12-bit data address - up to 4kB data memory (RAM)
+- 8-bit data bus
+
+Regerences:
+- https://ww1.microchip.com/downloads/en/DeviceDoc/31029a.pdf
+- https://developerhelp.microchip.com/xwiki/bin/view/products/mcu-mpu/8bit-pic/8-bitbl/8bitblis/
+- https://infonics.wordpress.com/wp-content/uploads/2015/03/introduction-to-pic-18-microcontrollers1.pdf
+- https://en.wikipedia.org/wiki/PIC_instruction_listings#Mid-range_core_devices_(14_bit)
+- https://ww1.microchip.com/downloads/en/DeviceDoc/MPLAB%20XC8%20PIC%20Assembler%20User%27s%20Guide%2050002974A.pdf
+
+"""
+
```

Comment:
Lmao love this ... Idc why its so exciting to have two arches emulated in a dumb way in a language that has no business implementing this, but it is! 🤣 :feelsgood: